newIDE/app/src/UI/Menu/ContextMenu.js

```diff
@@ -44,14 +44,17 @@ class MaterialUIContextMenu extends React.Component {
     return this.state.open ? (
       <Menu
         open={this.state.open}
-        anchorPosition={{ left: this.state.anchorX, top: this.state.anchorY }}
+        anchorPosition={{
+          left: this.state.anchorX,
+          top: this.state.anchorY,
+        }}
         anchorReference={'anchorPosition'}
         onClose={this._onClose}
         TransitionComponent={Fade}
         {...this.menuImplementation.getMenuProps()}
       >
         {this.menuImplementation.buildFromTemplate(
-          this.props.buildMenuTemplate()
+          this.props.buildMenuTemplate(this.props.i18n)
```

Comment:
Was this working? I've not seen i18n being passed to ContextMenu props. Anyway, let's *not* do that. Instead, "simply" add `<I18n>` around `<Menu` here in the MaterialUIContextMenu.

This allows to avoid adding `<I18n>` everywhere in the codebase : the context menu will pass it and does not need it as a prop :)